src/makeInfinityAdder.js

```diff
@@ -1,12 +1,26 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
Using `num === undefined` is correct, but you could also use `arguments.length === 0` to check if the function was called without arguments. This is a more explicit way to determine if no arguments were passed.